packages/quantic/force-app/main/default/lwc/exampleSearch/exampleSearch.html

```diff
@@ -15,7 +15,7 @@
                   <c-quantic-facet field="objecttype" label="Type" engine-id={engineId} data-cy="type"></c-quantic-facet>
                 </div>
                 <div class="slds-m-bottom_large">
-                  <c-quantic-facet field="filetype" label="File Type" engine-id={engineId}></c-quantic-facet>
+                  <c-quantic-facet display-values-as="link" field="filetype" label="File Type" engine-id={engineId}></c-quantic-facet>
```

Comment:
Does it impact the tests in `examples-search.cypress.ts`?

Reply:
they are all green from what i can see